apps/backend/src/modules/users/user.repository.ts

```diff
@@ -38,6 +38,11 @@ class UserRepository implements Repository {
 		return users.map((user) => UserEntity.initialize(user));
 	}
 
+	public async findByEmail(email: string): Promise<null | UserEntity> {
+		const user = await this.userModel.query().findOne({ email });
+		return user ? UserEntity.initialize(user) : null;
```

Comment:
```suggestion
		return user ? UserEntity.initialize({
		  email: user.email,
		  id: user.id,
		  ...
		}) : null;
```
For `initialize` you should enter each field separately